src/py/pyodide/__init__.py

```diff
@@ -17,20 +17,24 @@
 from typing import Any
 
 from . import _state  # noqa: F401
-from .code import CodeRunner  # noqa: F401
-from .code import eval_code  # noqa: F401
-from .code import eval_code_async  # noqa: F401
-from .code import find_imports  # noqa: F401
-from .code import should_quiet  # noqa: F401
-from .ffi import ConversionError  # noqa: F401
-from .ffi import JsException  # noqa: F401
-from .ffi import JsProxy  # noqa: F401
-from .ffi import create_once_callable  # noqa: F401
-from .ffi import create_proxy  # noqa: F401
-from .ffi import destroy_proxies  # noqa: F401
-from .ffi import register_js_module  # noqa: F401
-from .ffi import to_js  # noqa: F401
-from .ffi import unregister_js_module  # noqa: F401
+from .code import (
+    CodeRunner,  # noqa: F401
+    eval_code,  # noqa: F401
+    eval_code_async,  # noqa: F401
+    find_imports,  # noqa: F401
+    should_quiet,  # noqa: F401
```

Comment:
Looks like its parser is able to handle comments unlike isort

Reply:
This really should have an `__all__`, which would make the comments unneeded.

Reply:
We didn't want to put these in `__all__` because they are deprecated as exports.

Reply:
Actually according to our deprecation timeline these should be removed in 0.23.0, so we can take this stuff out.

Reply:
Yes, good point. Will do it in a separate PR so it's more easily discoverable as users will likely hit it it they haven't paid attention to console warnings.